src/components/Models/AnimationModel/AnimationModel.component.tsx

```diff
@@ -38,7 +39,7 @@ export const AnimationModel: FC<AnimationModelProps> = ({
     setAnimationRunning(false);
   };
 
-  const { scene } = useGltfLoader(modelSrc);
+  const { scene } = useLoader(GLTFLoader, String(modelSrc));
```

Comment:
We need the loader to be compatible with mesh compression and DRACO.

Take a look at our current loader on what to add:
https://github.com/readyplayerme/visage/blob/bd2fc81d29fecbfce2a8e17a1086fe46520174c2/src/services/Models.service.tsx#L214-L219

Reply:
nice, didnt see it